Tests/ContainerTests.swift

```diff
@@ -27,6 +27,14 @@ class ContainerTests: QuickSpec {
                     expect(container.playback?.pluginName) == "AVPlayback"
                 }
 
+                it("should has background color `clear`") {
```

Comment:
We don't need to necessarily change this right now, but keep in mind: using `should` in tests is a bad practice.

Quoting [from here](http://www.betterspecs.org/#should):

> Do not use should when describing your tests. Use the third person in the present tense.

Reply:
Ok, i will read this on future to improve my unit tests.